src/main/javadoc/overview.html

```diff
@@ -7,10 +7,10 @@
 
 The Jenkins git client plugin provides an API to execute general-purpose git operations on a local or remote repository.
 Its primary use is from the
-<a href="https://plugins.jenkins.io/git" target="_blank" unselectable="on">git plugin</a>;
+<a href="https://wiki.jenkins.io/display/JENKINS/Git+Plugin" target="_blank" unselectable="on">git plugin</a>;
```

Comment:
`plugins.jenkins.io` is actually fine for permalinks like this. It was just the use from `<url>` in the POM which was a problem, since that is precisely the metadata that `plugins.jenkins.io` uses to determine the content to embed!